### PR TITLE
mui v5 

### LINKED ISSUE #161: Support mui v5 setup for new apps
we'll want new apps already using that instead of needing changes/updates.

@kelarcrisp @markrawls

### PR DESCRIPTION
**Pull request by kelarcrisp**
closes #161 

new material-ui / emotion dependencies to work with MUI v5

### CODE REVIEW

File: zygoat/components/frontend/dependencies/resources/_app.js [hunk 1]
```diff
@@ -1,32 +1,19 @@
...
-}
+configure({ enforceActions: 'never' });
+enableStaticRendering(typeof window === 'undefined');
+
+const muiCache = createCache({
+  key: 'mui',
+  prepend: true,
+});
 
 const App = ({ Component, pageProps }) => {
```
Comment (saravenkatesh) [inline zygoat/components/frontend/dependencies/resources/_app.js:18]: naive q: the mui nextjs example passes in an `emotionCache` prop to `App` (and I think `_document.js` explicitly passes this prop when it calls `App`).  Should that be included here?

Comment (kelarcrisp) [reply zygoat/components/frontend/dependencies/resources/_app.js:18]: I see what you're saying. They're explicitly passing the cache for SSR and then default the value to another cache instance for the client side. And currently we're only creating the cache client side. Nice catch!

**Comment by kelarcrisp** [comment]
@markrawls this is probably a stupid question but do you see the problem the linter is yelling about in _app.js or in _document.js ? I don't see it :(

**Comment by markrawls** [comment]
Those are because your editor is using Windows style line endings and our prettier config demands UNIX style line endings. Pushed a commit to update that.

**Review by markrawls — APPROVED** [review]
LGTM! Great job on that.

**Pull request merged**